Cargo.toml

```diff
@@ -105,6 +105,16 @@ adler.debug = 0
 gimli.debug = 0
 miniz_oxide.debug = 0
 object.debug = 0
+rustc-demangle.debug = 0
+# Similar to above, as these libraries are used for backtrace symbolication,
+# we can afford to sacrifice a small amount of performance in order to get
+# smaller binaries by optimizing for size.
+addr2line.opt-level = 's'
+adler.opt-level = 's'
+gimli.opt-level = 's'
+miniz_oxide.opt-level = 's'
+object.opt-level = 's'
+rustc-demangle.opt-level = 's'
```

Comment:
hmm is there a strong reason to prefer this over making these like

```toml
rustc-demangle = { debug = 0, opt-level = 's' }
```

Reply:
That makes more sense, I just copied the existing pattern. (Pushed that change)